modules/apps/item-selector/item-selector-api/src/main/java/com/liferay/item/selector/ItemSelectorCriterion.java

```diff
@@ -60,6 +62,10 @@
 	 */
 	public List<ItemSelectorReturnType> getDesiredItemSelectorReturnTypes();
 
+	public default String getMimeTypeRestriction() {
+		return StringPool.STAR;
```

Comment:
Why not use here the `MIME_TYPE_RESTRICTION_DEFAULT` constant?

Reply:
I will, I forgot on this case